### ISSUE TITLE
Stat panel: No sparkline shown in small graphs

### ISSUE THREAD

**Issue by der-eismann** [grafana/grafana#21760]
**What happened**: I tried to convert some small graphs (2 cols wide, 3 rows high) from singlestat to stat panels, but I noticed that the sparkline is missing. Only when I expand the image to 3 cols wide the sparklines appear.

**What you expected to happen**: Stat graphs show their sparkline regardless of their size.

**How to reproduce it (as minimally and precisely as possible)**: Create a 2x3 single stat panel, notice the sparkline. Convert to stat, sparkline is missing.

Grafana 6.5 with singlestat:
![screenshot1](https://user-images.githubusercontent.com/10406217/73179965-add83300-4114-11ea-8144-75def7f55992.png)

Grafana 6.6 with one panel converted to stat:
![screenshot2](https://user-images.githubusercontent.com/10406217/73180071-d8c28700-4114-11ea-884a-7f6100a39c3a.png)

Grafana 6.6 with one panel converted to stat & extended to 3 rows (also the value font is quite small compared to singlestat):
![screenshot](https://user-images.githubusercontent.com/10406217/73180102-e546df80-4114-11ea-828b-a34e72a5c193.png)

**Anything else we need to know?**:

**Environment**:
- Grafana version: 6.6.0
- Data source type & version: Prometheus 2.15.2
- OS Grafana is installed on: Official docker container
- User OS & Browser: Fedora 31 & Chrome 79.0.3945.130

**Comment by marefr** [comment]
Reproduced here: https://play.grafana.org/d/DhhJBPyZk/21760-stat-no-sparkline-shown-in-small-panels?orgId=1

If I make my window a bit smaller in width the sparkline is not visible.

**Comment by torkelo** [comment]
This is by design, if the panel is too small it auto hides the graph, maybe the limit when it is hidden needs to be tweaked

**Comment by Rylon** [comment]
I had a similar experience upgrading today, would it be possible to either make this a feature toggle, so we can choose to always display the graph, even in small widgets please?

Information density is quite high on some of our existing dashboards, which is great, and I'd rather not have to make every widget much bigger to fit the same data on it as before as it'll reduce that information density.

**Comment by mcwanek** [comment]
We are experiencing the same issue after upgrade to Grafana v7.3 (from v6.7) last week. Disappointing to see a fix for this not accepted. 

"Small panels" is an arbitrary value, it seems the user should be able to choose what's best for their own use. 

We have high-density dashboards as well that I do not wish to migrate away from the SingleStat plugin. In our case we are using repeating panels based on a Prometheus query with automatic layout on 1 line, so I don't want to have to force each panel to be large enough to show the spark lines.

Please provide some sort of override for the spark lines to be displayed in the Stat plugin.

**Comment by sakarikl** [comment]
This affects us also. sadly I had to leave our stats now without spark line. That forces me to do a lot more work checking things separately when previously just a glance in monitor told me everything I needed to know.

**Comment by der-eismann** [comment]
Yeah, that's why we are staying on v7 for now... this breaks a lot of dashboards for us and there even was a fix in #28806. Really disappointing.

**Comment by adrian-tarau** [comment]
Same here ... sparkline is randomly displayed ... I could not find a rule when it is displayed or not. A configuration would be better...let the user decide when to hide (Auto, Always, None)

**Comment by sakarikl** [comment]
We ended up meking the stat panels so big that they show sparkline and then zooming with browser to get them to desired smaller size.

**Comment by Fuco1** [comment]
The sparkline doesn't show up even when I put the panel to cover the entire screen.

**Comment by davehayes** [comment]
This issue is also here for me. It's very frustrating, because the auto grid forces you to have stat panels of a minimum size or get no graph. I'm very very confused as to why providing a simple "always draw the graph" option is apparently so much of an issue. 

This plus the auto grid makes designing good looking panels very difficult.

**Comment by tarko** [comment]
Getting bitten by this during grafana 7 > 9 migration as well. New stat panels with height 2 hide the sparkline that was previously visible just fine using same-sized singlestat.

**Comment by rarsenault** [comment]
Just here to thumbs-up this one, way too far into Grafana 9 to go backwards; there exists a toggle to turn off sparkline, it would be nice if "leave it on" was also an option.

**Comment by The1hauntedX** [comment]
Sad to see [#28806](https://github.com/grafana/grafana/pull/28806) closed out.

Sure, there's workarounds that involve resizing your browser window. But we have a hard enough time getting users to use the variable selectors on our dashboards. I can't imagine trying to show them such a janky solution when the real solution was proposed 3 years ago and ignored.

**Comment by nosalan** [comment]
There should be an option to display the sparkline regardless of the tile size. I'd say this is a bug not a feature request.
